libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Queue/CSFNetwork.m

```diff
@@ -258,6 +258,11 @@ - (void)executeAction:(CSFAction *)action {
     // performed in duplicateActionInFlight: will match.
     action.enqueuedNetwork = self;
     
+    
+    if (contributeProgress) {
+        [self.progress resignCurrent];
+    }
+    
```

Comment:
Those lines got lost in the merge https://github.com/forcedotcom/SalesforceMobileSDK-iOS/commit/4a2c34ca645b90973a537723eae5d46a75aa3dd0